src/midifallback.c

```diff
@@ -18,6 +18,7 @@
 #ifdef _WIN32
 
 #include "doomtype.h"
+#include <stdio.h>
```

Comment:
Either before or after the all the existing ones so you're not mixing up includes with `" "` and `< >`.